src/ibex_bluesky_core/callbacks/reflectometry/_det_map.py

```diff
@@ -128,13 +148,13 @@ def stop(self, doc: RunStop, _md: dict[str, Any] | None = None) -> None:
             return super().stop(doc, _md)
 
         current_time = time.time()
-        for x, y in zip(self.x_data, self.y_data, strict=True):
+        for x, y in zip(self.x_data, self.y_data, strict=True):  # type: ignore
```

Comment:
why does pyright fail this? isn't y_data still just a `Variable`? this is more a question than a review comment.